spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -193,6 +193,39 @@
     end
   end
 
+  context 'with doc_auth_selfie_capture set to true' do
+    before do
+      allow(IdentityConfig.store).to receive(:doc_auth_selfie_capture).and_return({ enabled: true })
+    end
+
+    it 'proceeds to the next page with valid info, including a selfie image' do
+      perform_in_browser(:mobile) do
+        visit_idp_from_oidc_sp_with_ial2
+        sign_in_and_2fa_user(user)
+        complete_doc_auth_steps_before_document_capture_step
+
+        expect(page).to have_current_path(idv_document_capture_url)
+        expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_id'))
+
+        attach_images
+        attach_selfie
+        submit_images
+
+        expect(page).to have_current_path(idv_ssn_url)
+        expect_costing_for_document
+        expect(DocAuthLog.find_by(user_id: user.id).state).to eq('MT')
+
+        expect(page).to have_current_path(idv_ssn_url)
+        fill_out_ssn_form_ok
+        click_idv_continue
+        complete_verify_step
+        expect(page).to have_current_path(idv_phone_url)
+        visit(idv_document_capture_url)
+        expect(page).to have_current_path(idv_phone_url)
```

Comment:
what is the intention of these 2 lines? 🤔 

note: we already have tests that check the permissibility to go and back forth between steps. and there are also new rule changes in flight that allow back navigation  without being forced to go the most recently completed steps.

Reply:
That's a good point. I stole that part from the test above it (w/o the selfie). But since it's there, we probably don't need it here. And with the work you mentioned, it'll likely break in the near future. I'll remove those lines.

Reply:
Removed in [39002b8](https://github.com/18F/identity-idp/pull/9677/commits/39002b8757363b12dfc274d78e10d28e56ffcb9b).